src/model/PageType.h

```diff
@@ -13,11 +13,24 @@
 
 #include <XournalType.h>
 
+enum class PageTypeFormat {
```

Comment:
All-caps for enums isn't what you would usually use for C++ and looks rather ugly to read tbw